spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 
 require 'rubygems'
 require 'rspec'
-require 'rspec/autorun'
+require 'rspec/its'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.